src/ray/core_worker/task_manager.cc

```diff
@@ -238,7 +238,7 @@ std::vector<rpc::ObjectReference> TaskManager::AddPendingTask(
   // The language frontend is responsible for calling DeleteObjectRefStream.
   if (spec.IsStreamingGenerator()) {
     const auto generator_id = spec.ReturnId(0);
-    RAY_LOG(DEBUG) << "Create an object ref stream of an id " << generator_id;
+    RAY_LOG(INFO) << "Create an object ref stream of an id " << generator_id;
```

Comment:
is this debug -> info intended? we should revert?

Reply:
Yes, it has been reverted.